data/korath/korath missions.txt

```diff
@@ -257,3 +257,78 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Efret: Quarg Rescue"
+	boarding
+	repeat
+	invisible
+	deadline 1
+	source
+		government "Kor Efret"
+	on offer
+		"old efret rep" = "reputation: Kor Efret"
+		# bring it low enough that capping this will result in war
+		"reputation: Kor Efret" <?= 1
+		conversation
+			branch "unfriendly quarg" # if you're already at war with them
+				"reputation: Quarg" < 0
+			`The Quarg have brought reinforcements, probably in reaction to a distress call by the <ship>. They hail you, with barely contained anger in their voice. "Remove yourself from hence, and we may yet forgive your aggression. The Kor Efret are under our protection! Think not even about capturing yonder vessel."`
```

Comment:
```suggestion
			`The Quarg have brought reinforcements, probably in reaction to a distress call by the <ship>. They hail you, with barely contained anger in their voice. "Remove yourself, and we may yet forgive your aggression. The Kor Efret are under our protection! Think not even about capturing yon vessel."`
```

First is extraneous, so can just be cut to avoid the awkwardness.

Second is the more appropriate according to quick Google search ( https://www.shakespeareswords.com/Public/LanguageCompanion/ThemesAndTopics.aspx?TopicId=45 ).

Reply:
I don't know what's up in the air about anything?

I really like the "from hence," and it sounds a *lot* less archaic than "yonder" or "yon," to me.  But as for the latter yes if I'm doing it wrong I should definitely not be.